modules/cytoid/profile.py

```
@@ -4,8 +4,14 @@
 from core.utils.http import get_url
 
 
-async def cytoid_profile(msg: Bot.MessageSession, uid):
-    profile_url = 'http://services.cytoid.io/profile/' + uid
+async def cytoid_profile(msg: Bot.MessageSession, username):
+    if username:
+        query_id = username.lower()
+    else:
+        query_id = CytoidBindInfoManager(msg).get_bind_username()
```

Ruff (0.8.2)

Undefined name `CytoidBindInfoManager`

(F821)